README.md

```diff
@@ -5,6 +5,7 @@ using Java.
 
 ## Important!
 
+* Next to this documentation, please read the [general documentation](https://quedex.net/doc/api) of our WebSocket API.
```

Comment:
I don't think this should be the first point.

Reply:
Hmm, why? I've thought that this is something that should be read before this documentation.